### ISSUE TITLE
Bug with first list items wrapped in unneeded View

### ISSUE THREAD

**Issue by winkelsdorf** [andangrd/react-native-markdown-package#21]
Using a custom font, I discovered that the list items are not rendered the same way.

The following example markdown is given:
```markdown
* Test 1 ß
* Test 2 ß
* Test 3 ß
```

The first list item text is wrapped in another View which makes it ignore setting a custom font:
<img width="120" alt="Screen Shot 2019-05-01 at 12 36 18" src="https://user-images.githubusercontent.com/1413291/57014508-b4329080-6c10-11e9-8828-d47faa4c4382.png">

Clearly visible in RN Debugger:
<img width="1028" alt="Screen Shot 2019-05-01 at 12 55 45" src="https://user-images.githubusercontent.com/1413291/57014521-c0b6e900-6c10-11e9-9d76-485c3b495fef.png">

The text of the first list item is wrapped in another View which receives the style, the others list items are just text nodes.

**Comment by andangrd** [comment]
@winkelsdorf I couldn't replicate this issue,
it works fine on my end.
![image](https://user-images.githubusercontent.com/26213148/57760906-aad41880-7726-11e9-9a2e-b5d081976807.png)

**Comment by winkelsdorf** [comment]
@andangrd Thanks for your reply! It should be visible in the React Native Debugger when checking the view hierarchy. Let me try to get some sample code for you.. Currently I'm snowed under work.

**Comment by andangrd** [comment]
HI @winkelsdorf 
I'm hoping that this issue has fixed by the latest update,

<img width="1609" alt="Screen Shot 2019-10-08 at 12 01 18 AM" src="https://user-images.githubusercontent.com/26213148/66332786-b0453000-e95f-11e9-9e04-5146b3afaea5.png">

**Comment by winkelsdorf** [comment]
Sorry for the late reply, just had the time to test it again with v1.6.0.

It's still a little buggy when the custom font is *NOT* defined in `text` style, but `listItemText`:

<img width="194" alt="Screen Shot 2019-12-22 at 00 54 57" src="https://user-images.githubusercontent.com/1413291/71315092-b2466780-2455-11ea-9aec-b520a94599c0.png">

It's working for the last two.

Example markdown:
```markdown
- Not bold &
- **Manually set to bold &**
- Suddenly bold &
```

The styles:
```js
const MarkdownStyles = {
  text: {
  },
  paragraph: {
    fontFamily: 'MyCustomFont',
  },
  strong: {
    fontFamily: 'MyCustomFont',
    fontWeight: 'bold',
  },
  listItemText: {
    fontFamily: 'MyCustomFont',
    fontWeight: 'bold',
  },
};
```

Looks like the following happens:
The first line ignores `listItemText` completely. And has an issue with the ampersand sign.

**Comment by andangrd** [comment]
Hi @winkelsdorf 

This issue has been fixed in the latest version, 1.7.0
Below is the testing result, and please let me know if it still not working on your end..

![image](https://user-images.githubusercontent.com/26213148/74682843-c4e0e100-51f9-11ea-88b1-55b76813a872.png)

**Comment by winkelsdorf** [comment]
Hi @andangr!

Thank you for still helping me with this 👍 

Looks way better, but the first line still ignores the custom font (have a look at the `ß`):
<img width="199" alt="Screen Shot 2020-02-17 at 21 55 24" src="https://user-images.githubusercontent.com/1413291/74685957-58052100-51d0-11ea-9391-b98fadaa1bfa.png">

**Comment by andangrd** [comment]
hi @winkelsdorf 

this issue should be fixed in the latest version, [v1.8.0](https://github.com/andangrd/react-native-markdown-package/releases/tag/v1.8.0)
please let me know if you're still facing this bug.

thank you
